docs/leaderboard/index.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 1
+sidebar_class_name: sidebar-item-overview
+sidebar_label: Momento Leaderboard
+title: Momento Leaderboard Documentation
+description: Learn about Momento's purpose-built Leaderboard API
+pagination_next: null
+---
+
+## What is Momento Leaderboard?
+
+Momento Leaderboard is a purpose-built leaderboard that supports tens of millions of items and rapid ingestion/querying/updates. While many databases approximate leaderboards via general purpose data structures like sorted set or range queries, Momento Leaderboard is a first-class, full-fledged service that you can quickly and easily integrate with your application. 
+
+## Getting Started with Momento Leaderboard
```

Comment:
I think there is currently a server-side allowlist for this feature, so we will have to add a paragraph stating that users need to contact us to get added to that before they can use it. (and maybe show the error they will get if they try to use it w/o the allowlist?)

Reply:
Why are we documenting (with caveats) something that will be available to exactly one customer?